x/cvm/types/genesis.go

```diff
@@ -22,12 +25,23 @@ func DefaultGenesisState() *GenesisState {
 	}
 }
 
-// ValidateGenesis validates cvm genesis data.
-func ValidateGenesis(bz json.RawMessage) error {
-	var data GenesisState
-	err := ModuleCdc.UnmarshalJSON(bz, &data)
-	if err != nil {
-		return err
+// Validate performs validation of cvm genesis data.
+func (gs GenesisState) Validate() error {
+	if gs.GasRate > 100 {
+		return fmt.Errorf("failed to validate %s genesis state: GasRate must ???", ModuleName)
```

Comment:
:thinking:

Reply:
Yeah 😅 What should be the valid range of gas rate?

Reply:
anything above 1 should be fine I think